nibabel/streamlines/compact_list.py

```diff
@@ -0,0 +1,232 @@
+import numpy as np
+
+
+class CompactList(object):
+    """ Class for compacting list of ndarrays with matching shape except for
```

Comment:
Should be single line description (no wrap) followed by blank line and more detailed description.